build/AzurePipelinesTemplates/MUX-RunHelixTests-Job.yml

```diff
@@ -67,6 +74,74 @@ jobs:
       filePath: build\Helix\PrepareHelixPayload.ps1
       arguments: -Platform '$(buildPlatform)' -Configuration '$(buildConfiguration)'
 
+  - task: powershell@2
+    displayName: 'Create RunTestsInHelix-InteractionTests.proj'
+    condition:
+      ne('${{parameters.testSuite}}','NugetTestSuite')
+    inputs:
+      targetType: filePath
+      filePath: build\Helix\GenerateTestProjFile.ps1
+      arguments: -TestFile '$(artifactsDir)\drop\$(buildConfiguration)\$(buildPlatform)\Test\MUXControls.Test.dll' -OutputProjFile '$(Build.SourcesDirectory)\build\Helix\RunTestsInHelix-InteractionTests.proj' -JobTestSuiteName 'DevTestSuite' -TaefPath '$(taefPath)'
+
```

Comment:
Consider lifting this into a template step with the TestFile (and anything else that changes) as the parameters to simplify this.

Reply:
Not a bad idea, I'll do that.